api/http/indigo_service/indigo_http.py

```diff
@@ -206,13 +209,67 @@ def common_bits(
     return jsonapi.make_common_bits_response(result)
 
 
+@app.post(f"{BASE_URL_INDIGO}/render")
+def render(
+    request: jsonapi.RenderRequest,
+) -> Union[Response, FileResponse]:
+    compound, *_ = service.extract_compounds(compounds(request))
+    output_format = request.data.attributes.outputFormat
+    indigo_renderer = IndigoRenderer(indigo())
+    indigo().setOption(
+        "render-output-format", jsonapi.rendering_formats.get(output_format)
+    )
+    options = request.data.attributes.options
+    for option, value in options.items():
+        indigo().setOption(option, value)
```

Comment:
Have you tested, what happens if we also define `render-output-format` in options list? According to the code, it will rewrite the `output_format` defined in request. I believe we need to check if they are equal, and if not, raise an exception, to not misguide the userrs.

Reply:
Not yet, and actually I've been thinking about how to escape this situation at all. For example, set `render-output-format ` the obligatory option and the only way to choose the output format. But this is probably worse than the method that you suggested. Thank you, I'll add a new test and check for this case.